src/test/java/org/broadinstitute/consent/http/service/DatasetRegistrationServiceTest.java

```diff
@@ -862,7 +864,15 @@ private DatasetRegistrationSchemaV1 createRandomCompleteDatasetRegistration(User
     consentGroup.setAccessManagement(AccessManagement.CONTROLLED);
     consentGroup.setDataLocation(ConsentGroup.DataLocation.TDR_LOCATION);
     consentGroup.setDataAccessCommitteeId(new Random().nextInt());
-
+    schemaV1.setAssets(Map.of(
+        "workspace_id", "c7b96ac5-5568-441c-a3f4-2e82e45e3e6d", 
```

Comment:
Should I make all these values random? The other fields are set to random numbers and strings.

Reply:
I think this is fine as is.